test.dart

```diff
@@ -1,3 +1,29 @@
-void main() {
-  print('Hello, Nikita!');
-}
+
+
+
+
+void main() 
```

Comment:
pls reorginize project to look like this
![telegram-cloud-photo-size-2-5463238138005679347-y](https://github.com/user-attachments/assets/c3609003-99aa-4557-aca2-2f2d61c59daa)